src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -429,13 +428,20 @@ export default {
         }],
         paperOnly: true,
         isCorrected: filing.header.isCorrected || false,
-        comments: this.flattenComments(filing.header.comments)
+        comments: this.flattenAndSortComments(filing.header.comments)
       }
       this.filedItems.push(item)
     },
 
-    flattenComments (comments: any): Array<any> {
-      return (comments && comments.length > 0) ? comments.map(c => c.comment) : []
+    flattenAndSortComments (comments: any): Array<any> {
```

Comment:
Are the comments also an array?

Reply:
Maybe, or they could be null. I didn't want to go crazy with typing in this component because it's not fully TypeScript yet.

Reply:
100%, totally understand, every little bit helps!